### DISCUSSION TITLE
Updated Laravel to 8.x and now getting convert string error.

### DISCUSSION THREAD

**Discussion by junitobermudez** [spatie/laravel-sluggable#185] [Q&A]
I updated my application to Laravel 8.x today and I am now getting the following 500 error. Object of class Spatie\Sluggable\SlugOptions could not be converted to string Please let me know if there was something I missed when updating to 8.x.

**Comment by seriousjelly** [comment]
I am also getting this issue on 8.x using the examples in the documentation, did you figure this out?

```
public function getSlugOptions() : SlugOptions
{
    return SlugOptions::create()
        ->generateSlugsFrom('name')
        ->saveSlugsTo('slug');
}

**Reply by junitobermudez** [reply]
Unfortunately I have not figured it out yet. This is happening to me specifically locally. I launched to staging and it worked fine. The thing is there is no difference. I use Laravel Vapor for staging and production. Not sure what is different locally.

**Reply by felipeva** [reply]
Were you guys able to fix this?
